openhtf/__init__.py

```diff
@@ -83,10 +87,22 @@ class TestPhaseInfo(mutablerecords.Record(
   """
 
   @classmethod
-  def WrapOrReturn(cls, func):
+  def WrapOrCopy(cls, func):
+    """Return a new TestPhaseInfo from the given function or instance.
+    
+    We want to return a new copy so that you can reuse a phase with different
+    options, plugs, measurements, etc.
+    
+    Args:
+      func: A phase function or TestPhaseInfo instance.
+    
+    Returns:
+      A new TestPhaseInfo object.
+    """
     if not isinstance(func, cls):
-      func = cls(func, inspect.getsource(func))
-    return func
+      return cls(func, inspect.getsource(func))
+    # We want to copy so that a phase can be reused
+    return copy.copy(func)
```

Comment:
ugh, good call, it's not as simple as deepcopy... hm, since you just did the isinstance check, you know that func is an instance of TestPhaseInfo, so you could explicitly copy one level deep, which I *think* is enough?